### ISSUE TITLE
The /s/s/ page that was designed to be inside a frame needs to be updated slightly

### ISSUE THREAD

**Issue by ckjpn** [Tatoeba/tatoeba2#3147]
The /s/s/ page that was designed to be inside a frame needs to be updated slightly.

The reason is that since the page was designed, browsers, for security reasons, have made it so being within a frame means the user is not logged in.

If someone could do this, I'd appreciate it.

There are 2 ways to solve this problem.  Number one is perhaps the best way.  Number two would be very easy to do.

### 1. Insert target="_blank" into the link

 Insert the following in the link and leave the "Click the top sentence to go to tatoeba.org ..." line in at the bottom of the page.

target="_blank"

This will put the full page into a different tab or window, so logged-in members can add translations or leave comments.

In other words, insert it between...
THIS: <a
AND THIS:  href="/en/sentences/show/
So it will look like the following on the https://tatoeba.org/en/s/s/388848 .

<a target="_blank" href="/en/sentences/show/388848"

### 2. Or, Remove the following from the bottom of the page, so members won't try to click the link to add a translation.

<p>
<i>
Click the top sentence to go to tatoeba.org to translate it or leave a comment.
</i>
</p>

### Here is a sample use of these pages within a frame.
https://aitech.ac.jp/~english/rndaudio/

You can see what happens when the top sentence is clicked. The member seems to get logged out.

**Comment by jiru** [comment]
I can implement your proposed solution 1.

By the way, I am working on a [new API](https://api.tatoeba.org/unstable#?route=get-/unstable/sentences/-id-) you can already use to access the sentence data. You could load the sentence and translations with JavaScript only, no longer needing iframes nor the /s/s endpoint. It’s a little bit more work for you, but you get more freedom to display sentences the way you see fit, and you don’t need to ask the Tatoeba team every time you need to change the /s/s pages.

**Comment by jiru** [comment]
I implemented solution 1, it will be available after the next update of Tatoeba.
